auth-sample/src/main/res/layout/activity_main.xml

```diff
@@ -20,126 +20,69 @@
     under the License.
 -->
 <android.support.design.widget.CoordinatorLayout
-    android:id="@+id/activity_main"
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="com.spotify.sdk.android.authentication.sample.MainActivity">
 
-    <android.support.constraint.ConstraintLayout
```

Comment:
There was a crash related to ellipsizing the text, which could be fixed by removing that attribute, but then the layout would break (views would overlap). Since this is just a demo I decided to replace this with a simple LinearLayout instead of trying to fix the constraints on this layout.
More info in the commit message.